### ISSUE TITLE
is rhel7/fixes/ansible/service_avahi-daemon_disabled.yml needed?

### ISSUE THREAD

**Issue by shawndwells** [OpenSCAP/scap-security-guide#2724]
``rhel7/fixes/ansible/service_avahi-daemon_disabled.yml `` still exists, however ``avahi-daemon`` is disabled in the service_disabled templates. Ref:
https://github.com/OpenSCAP/scap-security-guide/blob/master/rhel7/templates/csv/oval_5.11/services_disabled.csv#L6

<snip>
`````
# service_name, package_name, daemon_name (as recognized by chkconfig / systemd. To be used when daemon_name differs from service_name)
....
avahi-daemon,avahi,
`````

Can we delete rhel7/fixes/ansible/service_avahi-daemon_disabled.yml ?

**Comment by redhatrises** [comment]
@shawndwells `rhel7/fixes/ansible/service_avahi-daemon_disabled.yml` disables the socket as well. The template does not attempt to disable the avahi socket.

**Comment by matejak** [comment]
This has been resolved as part of unshadowing, see #3055, #3025 and #3029.
